examples/multiomics_mogonet/main.py

```diff
@@ -0,0 +1,107 @@
+"""
+Multiomics Integration via Graph Convolutional Networks for Cancer Classification Tasks.
+
+Reference:
+Wang, T., Shao, W., Huang, Z., Tang, H., Zhang, J., Ding, Z., Huang, K. (2021). MOGONET integrates multi-omics data
+using graph convolutional networks allowing patient classification and biomarker identification. Nature communications.
+https://www.nature.com/articles/s41467-021-23774-w
+"""
+
+import argparse
+import warnings
+
+import pytorch_lightning as pl
+import torch
+from config import get_cfg_defaults
+from model import MogonetModel
+
+import kale.utils.seed as seed
+from kale.loaddata.multiomics_datasets import SparseMultiOmicsDataset
```

Comment:
Since we treat multiomics as one word, it should be `SparseMultiomicsDataset`

Reply:
Changed.